CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ For details about compatibility between different releases, see the **Commitment
 ### Added
 
 - Option to store rate limits in Redis. When used, the rate limits are applied over the entire cluster instead of per-instance.
+- Field `complement_collaborators` was added to `SearchAccounts`. Allows the user to request the complement of an entity's collaborators.
```

Comment:
What does "complement of an entity's collaborators" mean?

Reply:
Complement here would be a [set complement](https://en.wikipedia.org/wiki/Set_(mathematics)#/media/File:Venn1010.svg). I was wondering if I should've been more explicit and be written like:

```
- Field `complement_collaborators` was added to `SearchAccounts`. Allows an user to request the accounts that are not already attached to the entity's collaborator list.
```

Reply:
If you think that is best I'll replace the text then